node-fs-readfile/cat.js

```diff
@@ -0,0 +1,18 @@
+import { readFile } from 'node:fs/promises';
+
+const fileArray = process.argv.slice(2);
+
+async function print() {
+  await Promise.all(
+    fileArray.map(async (file) => {
+      try {
+        const data = await readFile(`./${file}`, { encoding: 'utf8' });
```

Comment:
As a shortcut, you can pass the encoding as a string to the second parameter. Also, since readFile will by default look in the current directory, it isn't necessary to prepend `./`. So you could write this more succinctly as
`const data = await readFile(file, 'utf8');`